packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -12,20 +47,108 @@ export default {
     name: 'ImageTile',
     components: {},
     props: {
+        href: {
+            type: String,
+            default: null
+        },
+        tileId: {
+            type: String,
+            default: null
+        },
+        isSelected: {
+            type: Boolean,
+            default: false
+        },
+        isLink: {
+            type: Boolean,
+            default: false
+        },
+        displayText: {
+            type: String,
+            default: null
+        },
+        imgSrc: {
+            type: String,
+            default: null
+        }
+    },
+    data () {
+        return {
+            isFilterSelected: false
```

Comment:
super NIT: mentioned in another comment - perhaps it's worth avoiding references to the word `filter` just to keep any domain references out of this component's concerns

Reply:
Yep, good spot 👍